sql/mozfun/ltv/android_states_v1/udf.sql

```diff
@@ -0,0 +1,62 @@
+/*
+  LTV states for Android. Results in strings like:
+  "1_dow3_2_1" and "0_dow1_1_1"
+*/
+CREATE OR REPLACE FUNCTION ltv.android_states_v1(
+  adjust_network STRING,
+  days_since_first_seen INT64,
+  submission_date DATE,
+  first_seen_date DATE,
+  pattern INT64,
+  active INT64,
+  max_weeks INT64,
+  country STRING
+) AS (
+    -- Users age. 0 on their first day (we need this uniquely ID'd) and then number of the week (1-indexed) since they were new.
```

Comment:
```suggestion
    -- Client's age. 0 on their first day (we need this uniquely ID'd) and then number of the week (1-indexed) since they were new.
```